GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -3227,15 +3227,22 @@ public void PopulateFavouriteRepositoriesMenu(ToolStripDropDownItem container, I
 
         private void FormBrowse_DragDrop(object sender, DragEventArgs e)
         {
-            CommitInfoTabControl.SelectedTab = TreeTabPage;
             HandleDrop(e);
         }
 
         private void HandleDrop(DragEventArgs e)
         {
+            if (TreeTabPage.Parent == null)
+            {
+                return;
+            }
+
+            var repoDir = Module.WorkingDir;
```

Comment:
This is only used in `IsFileExistingInRepo`, so probably worth inlining.

Reply:
Done.

One of the reason (and after refactoring the only remaining one) why I created this variable is to have a better naming.
I hate this `WorkingDir` name which I find too unclear.
Can I do a PR to rename it `RepoPath` (or `RepositoryPath`)?